src/Cake.Common/Tools/DotNetCore/Test/DotNetCoreTestSettings.cs

```diff
@@ -22,6 +22,11 @@ public sealed class DotNetCoreTestSettings : DotNetCoreSettings
         public DirectoryPath OutputDirectory { get; set; }
 
         /// <summary>
+        /// Gets or sets the XML output file
+        /// </summary>
+        public FilePath XmlPath { get; set; }
+
```

Comment:
@Grinderofl and we appreciate the PR.  In the meantime, you should be able to add this input parameter using the ArgumentCustomization option which is available on all Tools.